chain/chain/src/chain.rs

```diff
@@ -444,7 +444,12 @@ pub struct Chain {
     /// place in the database. Instead, we will include this "bonus changes" in
     /// the next block we'll be processing, keeping them in this field in the
     /// meantime.
-    pending_state_patch: Option<SandboxStatePatch>,
+    ///
+    /// Note that without `sandbox` feature enabled, `SandboxStatePatch` is
+    /// a unit type.  All methods of the type are no-ops which behave as if the
```

Comment:
```suggestion
    /// a ZST type.  All methods of the type are no-ops which behave as if the
```

That's the official term: https://doc.rust-lang.org/nomicon/exotic-sizes.html#zero-sized-types-zsts. 

It's jargony, but it's common enough to push to learning jargon.